python/astro_metadata_translator/bin/translateheader.py

```diff
@@ -88,7 +124,8 @@ def build_argparser():
 
 
 def read_file(file, hdrnum, dumphdr, quiet, print_trace,
-              outstream=sys.stdout, errstream=sys.stderr):
+              outstream=sys.stdout, errstream=sys.stderr, output_mode="verbose",
```

Comment:
It can't be auto because the decision on what auto means is done earlier. Also verbose was the previous default before today. Mainly it doesn't really matter because in real life this will only be called from process_files.